src/utils/api.js

```diff
@@ -24,6 +27,10 @@ export default class APIClient {
     }
     headers.append('X-CSRFToken', Cookie.get('csrftoken-20170707'));
```

Comment:
Is this still needed? Didn't your recent work on Normandy make csrf cookies obsolete?